### PR TITLE
feat(multiarch): Choosing correct multiarch golang image

### PR DESCRIPTION
**Pull request by ecktom**

### CODE REVIEW

**Comment by ecktom** [comment]
Allowing platform specific Dockerfiles
Resolving to eg gridx/arm32v7/xxxx in Docker Hub

File: images/golang-tools/pipeline.config.json [hunk 1]
```diff
@@ -2,12 +2,24 @@
...
     }
+  },
+  {
+    "steps": {
+      "build": {
+        "id": "build_docker_with_build_arg_arm",
+        "env": {
+          "TARGET_ARCH": "arm32v7",
+          "BUILD_ARG": "GO_VERSION",
+          "BUILD_ARG_VARIANTS": "1.10;1.11"
```
Comment (jhedev) [inline images/golang-tools/pipeline.config.json]: please use 1.10.4 instead of 1.10

Comment (ecktom) [reply images/golang-tools/pipeline.config.json]: There is no tag for resin/armv7hf-golang:1.10.4 ;D Would suggest to go with this one till we proceeded with the multiarch idea

Comment (ecktom) [reply images/golang-tools/pipeline.config.json]: Could offer you a 1.10.3 ;)

Comment (jhedev) [reply images/golang-tools/pipeline.config.json]: I guess 1.10.3 is fine then

Comment (jhedev) [reply images/golang-tools/pipeline.config.json]: or just leave the 1.10

Comment (ecktom) [reply images/golang-tools/pipeline.config.json]: went with 1.10.2 (acutally used in client repo) and 1.10.3

**Review by jhedev — APPROVED** [review]

**Pull request merged**